consensus/core/src/dag_state.rs

```diff
@@ -206,6 +218,26 @@ impl DagState {
         self.highest_accepted_round
     }
 
+    /// Index of the last commit.
+    pub(crate) fn last_commit_index(&self) -> CommitIndex {
+        match &self.last_commit {
+            Some(commit) => commit.index,
+            None => 0,
+        }
+    }
+
+    /// Last committed round per authority.
+    pub(crate) fn last_committed_rounds(&self) -> Vec<Round> {
+        match &self.last_commit {
+            Some(commit) => commit.last_committed_rounds.clone(),
+            None => vec![0; self.context.committee.size()],
```

Comment:
I am not sure there is a straightforward way to do this because the authority count is not known at compile time?

Reply:
Good point. Either this can be `vec![]` and this special case can be handled at callsite, or it seems the callsite expects a Vec so returning by value is fine for now.